webapp/package-lock.json

```diff
@@ -341,11 +341,6 @@
                   "dev": true
```

Comment:
Are the updates to this file required? Usually I would exclude local changes to this file from my commit